src/Tizen.NUI/src/public/Rendering/RendererConstants.cs

```diff
@@ -0,0 +1,465 @@
+        Screen = 0x9295,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Overlay = 0x9296,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Darken = 0x9297,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Lighten = 0x9298,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        ColorDodge = 0x9299,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        ColorBurn = 0x929A,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        HardLight = 0x929B,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        SoftLight = 0x929C,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Difference = 0x929E,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Exclusion = 0x92A0,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Hue = 0x92AD,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Saturation = 0x92AE,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Color = 0x92AF,
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Luminosity = 0x92B0
+    }
+
+    /// <summary>
+    /// Enumeration for blend factor.
+    /// </summary>
+    /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum BlendFactorType
+    {
+        /// <summary>
+        /// Match as GL_ZERO
+        /// </summary>
```

Comment:
All enum values have to get `[EditorBrowsable(EditorBrowsableState.Never)]` tag or ACR `version`.
It's WIP PR now, so it might not be added  yet.?
Same as below.

Reply:
I don't know detail but... Should we add that tags for each enum? For example, `NUIConstants.cs`'s `ResizePolicyType` mark since_tizen only at `enum` define, and not add tags for each values.

Reply:
AFAIK, all values should have each `since_tizen` version tag, but I found that some values have, but some values don't, as you said... :(

For example, the document for `LayoutParamPolicies` is like this :
https://docs.tizen.org/application/dotnet/api/TizenFX/API10/api/Tizen.NUI.BaseComponents.LayoutParamPolicies.html

On the other hands, the document for `States`, which does not write down its version, is like this :
https://docs.tizen.org/application/dotnet/api/TizenFX/API10/api/Tizen.NUI.BaseComponents.View.States.html

`LayoutParamPolicies` and `States` enums are all for `View` and they're included in `ViewEnum.cs`.
So, I think we need to get other NUI members' opinions for our unity.

Reply:
https://github.com/Samsung/TizenFX/pull/5165/commits/1a5ba365f3ce94601dd44c99868e94cbb6965104

Thank you for your comment. After some offline discussion, I decide to write `[EditorBrowsable(EditorBrowsableState.Never)]` tag for all properties.

Reply:
Thank you for adding it all. :)
I'll check it again.